### PR TITLE
PSMDB-1450 Release notes 6.0.15-12

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   docs/release_notes/6.0.15-12.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/release_notes/6.0.15-12.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Improvements and bug fixes, provided by MongoDB and included in Percona Server for MongoDB are the following:
+
+* [SERVER-72703](https://jira.mongodb.org/browse/SERVER-72703) - Changed the requirement to use exclusive write lock to intent exclusive write lock that doesn’t prevent reading from a collection during the $out stage when running the rename collection command.
+* [SERVER-78556](https://jira.mongodb.org/browse/SERVER-78556) - Fixed the issue with the replication lag by changing the `internalInsertMaxBatchSize` default value to 64.
+* [SERVER-83602](https://jira.mongodb.org/browse/SERVER-83602) - Fixed the issue with the match expression optimization for the $or to an $in rewrite by avoiding creating directly nested $or.
+* [SERVER-80363](https://jira.mongodb.org/browse/SERVER-80363) - Explicitly stated that the missing w field from write concern object will be filled with default write concern value
+
+Find the full list of new features and improvements in the release notes for [MongoDB 6.0.15 Community Edition](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.15---apr-18--2024).
+
+
```
Comment (igorsol) [inline docs/release_notes/6.0.15-12.md:32]: PSMDB-1418 should be mentioned in these release notes

**Review by igorsol — CHANGES_REQUESTED** [review]

**Review by igorsol — APPROVED** [review]
LGTM

File: docs/release_notes/6.0.15-12.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Percona Server for MongoDB 6.0.15-12 (2024-04-30)
+
+[Installation](../install/index.md){.md-button}
+
+
+Percona Server for MongoDB 6.0.15-12 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB Community Edition 6.0.14
```
Comment (sandraromanchenko) [inline docs/release_notes/6.0.15-12.md]:
```suggestion
fully-compatible, drop-in replacement for MongoDB Community Edition 6.0.15
```

File: docs/release_notes/6.0.15-12.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Release Highlights
+
+A number of issues with sharded multi-document transactions in sharded clusters of 2 or more shards have been identified that result in returning incorrect results and missing reads and writes. The issues occur when the transactions’ metadata is being concurrently modified by using the following operations: `moveChunk`, `moveRange`, `movePrimary`, `renameCollection`, `drop`, and `reshardCollection`. 
+
+The data is affected when using the following functionalities:
+
+* Sharded multi-document transactions
+* Queryable encryption
+
+The issues are fully fixed in MongoDB  4.4.29, 5.0.25, 6.0.14, 7.0.8, and are included in Percona Server for MongoDB 4.4.29-28, 5.0.26-22,6.0.14-11 and 7.0.8-5. If you are using sharded multi-document transactions or queryable encryption, upgrade to Percona Server for MongoDB 7.0.8-5 as soon as possible. Please follow closely the upstream recommendations for mitigation and remediation outlined in [Sharded multi-document transactions may perform operations using inconsistent sharding metadata](https://www.mongodb.com/alerts/critical-alert-sharding-txn-issues-apr-2024) alert.
```
Comment (sandraromanchenko) [inline docs/release_notes/6.0.15-12.md]:
```suggestion
The issues are fully fixed in MongoDB  4.4.29, 5.0.25, 6.0.14, 7.0.8, and are included in Percona Server for MongoDB 4.4.29-28, 5.0.26-22,6.0.14-11 and 7.0.8-5. If you are using sharded multi-document transactions or queryable encryption, upgrade to Percona Server for MongoDB 6.0.14-11 or later versions as soon as possible. Please follow closely the upstream recommendations for mitigation and remediation outlined in [Sharded multi-document transactions may perform operations using inconsistent sharding metadata](https://www.mongodb.com/alerts/critical-alert-sharding-txn-issues-apr-2024) alert.
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**